docs/AddingDoppl.md

```diff
@@ -0,0 +1,565 @@
+
+- In the project options pane of the new-project wizard, fill in `iosApp`
+for the "Product Name". Choose Swift as the programming language.
+Fill in the rest of the fields as you see fit, for
+whatever you want for your project. Then, click Next.
+
+![Xcode New-Project Wizard, Second Page Form](./AddingDoppl-6.png)
+
+- In the Finder-style directory-picker dialog that appears, choose your
+Android Studio project's root directory, make sure that the "Create Git repository"
+checkbox is unchecked, then click OK to close the dialog.
+
+After several seconds of churning, you should then be taken to an empty
+Xcode project window.
+
+![Xcode Project Window](./AddingDoppl-7.png)
+
+At this point, choose File > Close Project from the main menu,
+to close up that IDE window.
+
+Then, repeat these steps, but using `iosTest` instead of `iosApp` as the product
+name. You can use File > New > Project... from the main menu to start creating the
+`iosTest` project.
+
+## Step #9: Add the Pod to the Projects
+
+Open up your favorite text editor. Then, take the following content and put
+it in a file named `Podfile` in the `iosApp/` directory that you created
+in the previous step:
+
+```ruby
+platform :ios, '9.0'
+
+target 'iosApp' do
+    use_frameworks!
+    pod 'doppllib', :path => '../app/build'
+end
+```
+
+Here, 'iosApp' is the name of our newly-created Xcode project, and `:path`
+points to the `app/build/` directory of the Android Studio project, relative
+to the `iosApp/` directory. If you elected to put `iosApp/` somewhere else, you
+will need to adjust the `:path` value to the proper relative path.
+
+Then, create another `Podfile`, this time in `iosTest/`, where the `target`
+is `iosTest`:
+
+```ruby
+platform :ios, '9.0'
+
+target 'iosTest' do
+    use_frameworks!
+    pod 'testdoppllib', :path => '../app/build'
+end
+```
+
+Open up a command prompt on your iOS development machine (Terminal, iTerm2,
+etc.). Navigate to the `iosApp/` directory, and run `pod install`. This will
+take what Doppl generated and set it up as a pod for use in this Xcode project.
+Then, switch to the `iosTest/` directory and run `pod install` there. Each
+should give you output akin to the following:
+
+![Output of pod install Command](./AddingDoppl-8.png)
+
+## Step #10: Build the Pods
+
+Double-click on the `iosApp.xcworkspace` file in your `iosApp/` project or otherwise
+arrange to open it in Xcode. Once there, run the project. While this will not
+generate much useful output in the iOS Simulator, it will compile the pod that
+we defined in the preceding step. You will know when it is done when the number
+of build errors stops climbing, as there is no other obvious signal given by
+Xcode. In particular, a "Build Succeeded" popup will appear shortly before
+the build has actually completed.
+
+Then, repeat this process with the `iosTest.xcworkspace` file in the `iosTest/` project.
+
+## Step #11: Copy Some Files Into XCode
```

Comment:
Header could be more descriptive about what files are being copied. 

Less important: maybe even more explicitly section off the "app" part of the tutorial from the "testing" part. Though I see how you're trying to make it a linear series of steps so sections for "app" and "testing" may not make sense.

Reply:
Ummm... well, I changed it to "Copy Generated Files Into Xcode". The official reason &mdash; "Copy Files Into Xcode That I Was Told To Copy There To Make Things Work" &mdash; seems kinda wordy. :-) I'm up for other suggestions!